docs/devguides/contribution_guide.rst

```diff
@@ -2,43 +2,47 @@
 Contribution Guide
 =====================
 
-We are happy when you share your research with us and when you improve our
-code! There is an easy way to contribute to our project, follow the steps
-below. Your commit will be reviewed and added to our dev branch, and will be
-added to master branch with the nearest release. Moreover, if you are a
-dedicated contributor, you have a chance to get our t-shirt, get invited to
-one of our events or even join our team ; )
+We are happy that you share your research with us and want to improve our code!
+
+Please follow the steps below to contribute to our project.
+
+If you have any questions or suggestions about the contributing process,
+please share them with us on the `forum <https://forum.deeppavlov.ai>`_.
+Please note that we do not answer general questions in the github issues interface.
+
+If you are a regular contributor in the DeepPavlov open source project,
+you can receive an invitation to one of our events or an opportunity to become a part of our team.
 
 How to contribute:
 
-#. Don't start the coding first. You should **post an**
-   `issue <https://github.com/deepmipt/DeepPavlov/issues>`_ to discuss the
-   features you want to add. If our team or other contributors accept your offer
-   or give a +1, assign the issue to yourself. Now proceed with coding : )
+#. Don't start the coding first.
+   You should do a quick search over `existing issues <https://github.com/deepmipt/DeepPavlov/issues?q=is%3Aissue>`_
+   for the project to see if your suggestion was already discussed or even resolved.
+   If your issue doesn't already exist, please create a new one and state what exactly you would like
```

Comment:
I think `If your issue doesn't already exist` sounds bad: of course it doesn't exist since I haven't created it. Maybe It's better to rephrase with something like `issue describing you suggestion/problem` or `similar to what you are about to create`.

Reply:
`If nothing relevant found` probs because we just asked the contributor to scan the GH for relevant issues.